packages/@lwc/engine-core/src/framework/utils.ts

```diff
@@ -98,3 +100,20 @@ export function flattenStylesheets(stylesheets: TemplateStylesheetFactories): St
     }
     return list;
 }
+
+// Set a ref (lwc:ref) on a VM, from a template API
+export function setRefVNode(vm: VM, ref: string | undefined, vnode: VBaseElement) {
```

Comment:
`| undefined` seems weird here... why not guarding the call to it?

Reply:
It's 6 of one, half a dozen of the other to me... we can move the `ifUndefined` calls to outside this function. 🙂